Command/MonitorCommand.php

```diff
@@ -103,6 +103,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return Command::SUCCESS;
     }
 
+    // https://github.com/Dukecity/CommandSchedulerBundle/issues/99
+    private function formatDiff(\DateTime $dateTime): string
+    {
+        if($this->dateTimeFormatter)
+        {
+           return $this->dateTimeFormatter->formatDiff($dateTime);
+        }
+
+        return '';
```

Comment:
Why not returning a date string in this case ?

`$datetime->format('Y-m-d')`

Reply:
That would be duplicate the info in this format twice or not?